### DISCUSSION TITLE
RAM was set only 8 Gb.

### DISCUSSION THREAD

**Discussion by ibb5510** [microsoft/WSL#6030] [General] [answered]
[ubuntu-memory-2020-10-04.pptx](https://github.com/microsoft/WSL/files/5324775/ubuntu-memory-2020-10-04.pptx)
The computer has 128 Gb RAM, but wsl or unbutu uses 8 GB. Using stress to test wsl2-ubuntu performance, memory is totally used only 16 GB. Do we can fix the bug?

OS: Ubuntu 20.04.1 LTS on Windows 10 x86_64
 Kernel: 4.19.128-microsoft-standard
Uptime: 16 mins
 Packages: 643 (dpkg)
 Shell: bash 5.0.17
Terminal: /dev/pts/0
CPU: Intel i9-9900K (16) @ 3.600GHz
GPU: bdf4:00:00.0 Microsoft Corporation Device 008e
Memory: 109MiB / 7958MiB

**Comment by 0xbadfca11** [comment, marked as answer]
Not bug.
https://docs.microsoft.com/en-us/windows/wsl/release-notes#build-20175
https://docs.microsoft.com/en-us/windows/wsl/wsl-config#configure-global-options-with-wslconfig

**Comment by therealkenc** [comment]
In `%USERPROFILE%\.wslconfig`
```
[wsl2]
memory=128GB
```
Leaving a little headroom, so maybe `memory=96GB`, is not a bad idea.
